### PR TITLE
Milestone: Project finished

### PR DESCRIPTION
**Pull request by lordksix**
# Milestone: Project finished
In this milestone the project was finished:
- Added GitHub Actions workflows to automate linter.
- Copied files from Project-3, **check README.md** for more details.

### CODE REVIEW

**Comment by lordksix** [comment]
Hi @lordksix ,


Great job so far! 👏 
It shows that you worked really hard on this project, and you should be proud, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
- There are linter errors :pensive: 
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    
+    <footer>
+        <p>
+          ** New prototype of a portafolio of of microverse projects
+        </p>
+        <p>
+          &copy; Created by
+          <a href="https://github.com/lordksix" target="_blank"
+            >lordksix and Edcaly <i class="fab fa-free-code-camp"></i
+          ></a>
```
Comment (lordksix) [inline index.html:61]: Good job setting up live and useful links using <a> tag :+1: , however, whenever **target="_blank"** it is used, it is necessary to use **rel="noopener"**. More information can be found [here](https://webhint.io/docs/user-guide/hints/hint-disown-opener/). :eyeglasses:

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+  grid-gap: 4rem;
+  width: 100%;
+  max-width: 1280px;
+  margin: 0 auto;
+  margin-bottom: 6rem;
+}
+@media(max-width: 480px) 
+{
```
Comment (lordksix) [inline styles.css:30]: Awesome job! Very detailed stylesheet, however, There should always be a line between the end of the rule and the beginning of a new rule, and the opening bracket should always be on the same line as the opening of the rule

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+}
+body {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 1.8rem; /* 18px */
+  font-weight: 400;
+  line-height: 1.4;
+  background-color: red;
+  min-height: 100vh;
+}
+.nav {
```
Comment (lordksix) [inline styles.css:68]: Awesome job! Very detailed stylesheet, however, There should always be a line between the end of the rule.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  grid-gap: 4rem;
+  width: 100%;
+  max-width: 1280px;
+  margin: 0 auto;
+  margin-bottom: 6rem;
+}
+@media(max-width: 480px) 
+{
+  #h1
+  {
```
Comment (lordksix) [inline styles.css:32]: Awesome job! Very detailed stylesheet, however, the opening bracket should always be on the same line as the opening of the rule

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+      font-size: 1.8em;
+  }
+}
+
+#projects
+{
```
Comment (lordksix) [inline styles.css:38]: Awesome job! Very detailed stylesheet, however, the opening bracket should always be on the same line as the opening of the rule

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @lordksix,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Very polite comments. ✔️
 - Great use of inline comments. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**